community/contributing/nightly-builds/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Nightly Builds
+layout: single
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+Bugs affecting more than one service commonly surface only when the whole system is running. Our continuous integration system builds and runs a suite of integration tests against Spinnaker nightly on real cloud provider infrastructure to detect these bugs.
+
+> **Access the CI system here**: [https://builds.spinnaker.io](https://builds.spinnaker.io)
+>
+> Requires membership in the `build-cops` GitHub Team to access the system.
+
+## Build Cop
```

Comment:
Should this bit go in the governance repo?

Reply:
Is this related to governance, though? We were doing build cop rotations long before the governance model was adopted.

I put it here in between "Submitting a Patch" and "Releasing a Patch" because I assume a few nightly builds will pass between those two processes.

Reply:
Defining who we allow to hold the role seems like a governance thing to me. Are we letting any Approver take a rotation? Only select folks? Maybe it's tangential 🤷‍♂. Just a thought.

Reply:
Ahh, I see - you're saying the TOC should be in control of who is included in the `build-cops` rotation? Yes, the TOC seems like the appropriate place for this. 

**Corollary: It is also the responsibility of the TOC to recruit for this list.**